### ISSUE TITLE
Results on FSC147

### ISSUE THREAD

**Issue by jerpelhan** [niki-amini-naieni/CountGD#17]
I've run few versions of CountGD on the original FSC147 dataset without text-prompting and obtained the following results for the validation and test sets. I’d like for you to confirm these values.

| Model                         | Validation Set |          | Test Set    |          |
|-------------------------------|----------------|----------|-------------|----------|
|                               | MAE            | RMSE     | MAE         | RMSE     |
| CountGD_notxt_noSamTT         | 8.53           | 40.03    | 9.69        | 90.54    |
| CountGD_notxt                 | 8.12           | 38.97    | 8.35        | 89.80    |
| CountGD_notxt_no_SamTT_noCrop | 9.32           | 51.72    | 10.78       | 99.51    |

**Comment by niki-amini-naieni** [comment]
I just confirmed the result for the test set of CountGD_notxt_no_SamTT_noCrop. I have not had time to check the others yet, but will get back to you as soon as I can.

**Comment by jerpelhan** [comment]
Any updates?

**Comment by niki-amini-naieni** [comment]
Yes, I get the same results +- 0.01 (insignificant differences likely due to different hardware and libraries)!
